packages/medusa/src/api/routes/admin/uploads/get-download-url.ts

```diff
@@ -0,0 +1,34 @@
+import { IsString } from "class-validator"
+import { AbstractFileService } from "../../../../interfaces"
+import { validator } from "../../../../utils/validator"
+
+/**
+ * [get] /uploads
+ * operationId: "GetUploadsFileDownloadUrl"
+ * summary: "Gets a presigned download url for a file"
+ * description: "Gets a presigned download url for a file"
+ * x-authenticated: true
+ * parameters:
```

Comment:
suggestion: My bad - missed this in prev. review. Should this actually not be`requestBody`? 🤔 
```suggestion
 * requestBody:
 *   content:
 *     application/json:
 *       schema:
 *         required:
 *           - file_key
 *         properties:
 *           file_key:
 *             description: "key of the file to obtain the download link for"
 *             type: string
```